ExampleUsage/EnumExample.cs

```diff
@@ -99,6 +141,14 @@ public enum Size
 	public enum Shape
 	{
 		Square,
-		Round
+        [System.ComponentModel.Description("Round it is!")]
+        Round
 	}
+
+    public class Enum
```

Comment:
Then you would need to change the protection level of the class LookupValue. But that could be a good idea.